assets/src/scss/layout/_navbar.scss

```diff
@@ -12,7 +12,7 @@ $navbar-default-height: 60px;
     color: $white;
     fill: $white;
     font-family: $roboto;
-    font-size: $font-base-size;
+    font-size: $font-size-sm;
```

Comment:
Good catch, this `$font-base-size` was using `px` instead of `rem` our other sizes use. This likely caused issues when zooming.

We should probably do the same change in the 2 remaining places that use this variable, and remove it.
https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/layout/navbar/_search.scss#L88-L88
https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/layout/navbar/languages.scss#L6-L6